projects/elements/src/lib/lazy-elements/lazy-elements.module.ts

```diff
@@ -3,15 +3,18 @@ import {
   Optional,
   Inject,
   ModuleWithProviders,
-  InjectionToken
+  InjectionToken,
+  ANALYZE_FOR_ENTRY_COMPONENTS,
+  Type
 } from '@angular/core';
 import { CommonModule } from '@angular/common';
 
 import { LazyElementDirective } from './lazy-element/lazy-element.directive';
 import { LazyElementDynamicDirective } from './lazy-element-dynamic/lazy-element-dynamic.directive';
 import {
   LazyElementsLoaderService,
-  ElementConfig
+  ElementConfig,
+  LAZY_ELEMENT_ROOT_OPTIONS
```

Comment:
Maybe it  would be better to extract all the tokens into separate file `lazy-elements.tokens.ts` that way there will be no circular dependencies and no forward refs